specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-03-02-preview/managedClusters.json

```diff
@@ -5802,6 +5806,16 @@
       },
       "description": "Azure Key Vault key management service settings for the security profile."
     },
+    "ManagedClusterSecurityProfileWorkloadIdentity": {
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
```

Comment:
Better to use enum types, otherwise the arm reviewer is likely to push back.

Reply:
This enabled is the same as other addon feature. It has only enabled/disabled state. So I think using a boolean should be fine.